peakina/readers/parquet.py

```diff
@@ -0,0 +1,32 @@
+"""
+Module to enhance pandas.read_json with JQ filter
+"""
+
+from functools import wraps
+from typing import TYPE_CHECKING
+
+import pandas as pd
+import pyarrow.dataset as ds
+
+if TYPE_CHECKING:
+    from os import PathLike
+
+    FilePathOrBuffer = str | bytes | PathLike[str] | PathLike[bytes]
+
+
+@wraps(pd.read_parquet)
+def read_parquet(
+    path_or_buf: "FilePathOrBuffer",
+    preview_offset: int = 0,
+    preview_nrows: int | None = None,
+    columns: list[int] | None = None,
+) -> pd.DataFrame:
```

Comment:
```suggestion
    columns: list[int] | None = None,
    **kwargs
) -> pd.DataFrame:
```

I believe this should be added to ignore extra args, which can happen if the type of a datasource is modified

Reply:
Ok, I applied it. It triggers a lint test `Function is missing a type annotation`. How do I deal with this? 
That's completely uncharted territory for me.

Reply:
[fe3ea55](https://github.com/ToucanToco/peakina/pull/910/commits/fe3ea55e94e59d81cf584377ee40409c158e0dbb)